.eslintrc.json

```diff
@@ -54,6 +60,8 @@
         "react/jsx-filename-extension": "error",
         "react/prefer-stateless-function": "warn",
         "react/sort-comp": "warn",
-        "react/jsx-max-props-per-line": [1, { "when": "multiline" }]
+        "react/jsx-max-props-per-line": [1, { "when": "multiline" }],
+        "import/namespace": "off",
+        "import/default": "off"
```

Comment:
If these are necessary for jest (ideally they would not), I would actually like to have still these useful warnings enabled for the rest of the code. Could we have a separate linter configuration file that is used for the test files? Might be nice anyway, as e.g. jest globals are not acceptable in source code. The test config could extend the normal config and add all the jest-specific things. Then we specificy `**/*.test.js*` in the arguments to eslint while pointing it at the new config file.